infra.bs

```diff
@@ -675,10 +675,12 @@ if the following steps return true:
 
   <ol>
    <li><p>Let <var>aCodeUnit</var> be the <var>i</var>th <a>code unit</a> of <var>a</var> if
-   <var>i</var> is less than <var>a</var>'s <a for=string>length</a>; otherwise null.
+   <var>i</var> is less than <var>a</var>'s <a for="JavaScript string">length</a> as a
+   <a>JavaScript string</a>; otherwise null.
```

Comment:
Please remove "as a JavaScript string"; as discussed this just confuses matters.